ur_moveit_config/config/kinematics.yaml

```diff
@@ -1,4 +1,4 @@
-ur_manipulator:
+ur5e_manipulator:
```

Comment:
I don't think this change is sensible... This brakes all other UR models.

Reply:
Yep, I don't expect it to get merged like this but it is some last-minute debugging.

Reply:
I corrected this now

Reply:
FYI this solved this error:

`Error:   Semantic description is not specified for the same robot as the URDF`